lib/chef-dk/dist.rb

```diff
@@ -0,0 +1,31 @@
+module ChefDK
+  class Dist
+    # This class is not fully implemented, depending on it is not recommended!
+
+    # The full marketing name of the product
+    PRODUCT = "Chef Developmet Kit".freeze
```

Comment:
This might be an intended spelling, but just in case...
```suggestion
    PRODUCT = "Chef Development Kit".freeze
```